style.css

```diff
@@ -127,3 +127,86 @@ button:hover {
 .button-2 {
   margin-top: 25px;
 }
+
+.contact-main {
+  width: 100%;
+  border-top-left-radius: 120px;
+  background-color: #6070ff;
+  background-image: url(./images/contact-shape.svg);
+  background-repeat: no-repeat;
+  background-size: 70% 100%;
+  background-position-x: right;
+}
+
+.contact-me {
+  width: 92%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  padding-top: 160px;
+  margin: auto;
+}
+
+.text-contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.text-contact #contact-img {
+  padding-bottom: 20px;
+  width: 80%;
+  height: auto;
+  margin: auto;
+}
+
+.form {
+  width: 100%;
+  margin-top: 60px;
+}
+
+.form ul {
+  display: grid;
+  grid-template-rows: auto;
+  row-gap: 15px;
+  justify-content: center;
+  margin: auto;
+}
+
+input[type=text] input[type=email] {
+  width: 88vw;
+  height: 48px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 8px;
+}
+
+textarea {
+  width: 88vw;
+  height: 170px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin: auto;
+  padding: 8px;
+}
```

Comment:
- [ ] Great job so far 👍, however, the styling of your form is not quite there yet, here are the screenshots:
Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154129444-5201d85f-1509-4367-999e-117cf0f917bb.png) | ![image](https://user-images.githubusercontent.com/59416136/154129569-f985a8f4-5155-43c2-ab0e-e35e2f37e5ac.png)


Kindly fix this.